src/pages/_app.tsx

```diff
@@ -1,63 +1,39 @@
-import { WagmiConfig, createClient, configureChains } from 'wagmi';
-import { ChakraProvider, useColorMode } from '@chakra-ui/react';
+import { WagmiProvider, http } from 'wagmi';
+import { ChakraProvider } from '@chakra-ui/react';
 import type { AppProps } from 'next/app';
 import { theme } from '@/layouts/theme';
 import Layout from '@/layouts/Layout';
-import { publicProvider } from '@wagmi/core/providers/public';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import { defaultChain } from '@/utils/contract';
 import '@rainbow-me/rainbowkit/styles.css';
-import {
-  getDefaultWallets,
-  RainbowKitProvider,
-  lightTheme,
-  darkTheme,
-} from '@rainbow-me/rainbowkit';
-
-const { chains, provider } = configureChains(
-  [defaultChain],
-  [
-    publicProvider(),
-  ],
-);
-
-const { connectors } = getDefaultWallets({
-  appName: process.env['NEXT_PUBLIC_WALLET_CONNECT_PROJECT_NAME '] || '',
-  projectId: process.env['NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID '] || '',
-  chains,
-});
-
-const wagmiClient = createClient({
-  autoConnect: true,
-  connectors: connectors,
-  provider,
+import { getDefaultConfig } from '@rainbow-me/rainbowkit';
+import { RainbowKitProvider } from '@rainbow-me/rainbowkit';
+
+const config = getDefaultConfig({
+  appName: 'cit-web3-class',
+  projectId: '1b5b1d5c351c838062b62002e37bee3c',
```

Comment:
Is the projectId value fixed?
Also, what do you need it for?

Reply:
@HokutoMorita 

I am so sorry for the late reply,
the `Rainbow Kit` library needs project ID from `WalletConnect` for integration. The project ID is fixed for a DAPP so we can keep as it is in the source code.

Reply:
I see
Thank you!